nav.css

```diff
@@ -0,0 +1,102 @@
+* {
+	box-sizing: border-box;
+}
+body {
```

Comment:
OK, na początek musimy zaprowadzić porządek w kodzie. W naszych stylach selektor `body` występuje w kilku miejscach - mało tego w trzech ustawiamy `margin: 0` oraz `padding: 0`, a w dwóch `font-family: 'Oxygen', sans-serif;`. Oczywiście każdą właściwość wystarczy zadeklarować raz :) te deklaracje oprócz tego miejsca pojawiają się jeszcze w plikach `reset.css` oraz `style.css`. Oczywiście uznałbym, że tutaj czyli w `nav.css` nie musimy zamieszczać stylowania dotyczącego tagu `body`, więc stąd na pewno bym to skasował. Co do `margin:0` i `padding: 0` robimy to, żeby ujednolicić tak jakby wyjściowe style dla elementu `body` ponieważ niektóre przeglądarki potrafią dodawać własne bazowe style do niektórych elementów (np. `input` bez naszych styli jest szary i prostokątny, ma border itp. - podobnie `body` czasami dostaje padding i margin), takie rzeczy resetujemy w pliku `reset.css` i tam bym zostawił ten `margin` i `padding`. Po drugie deklarujemy tutaj też `font-family`, który jest już całkiem specyficzny dla naszej strony, więc to już w pliku `reset.css` znajdować się nie powinno. I tą deklarację umieściłbym w pliku `style.css`.